src/resources/postcss/tribe-events-full.pcss

```diff
@@ -1422,8 +1422,17 @@ select.tribe-events-dropdown {
 			}
 		}
 	}
+
+	body.admin-bar & {
```

Comment:
It can even precede like is done here. Since it precedes the output CSS rule is `body.admin-bar .datepicker`, thus it will only target the datepicker when the admin bar is present.

The advantage of doing it this way is that is that now we only have one `.datepicker` selector and all of the CSS pertaining to it is grouped together. If we ever change the name of that selector we only have to change it in one place. And, actually it probably does make sense to change it to the more verbose `.datepicker-dropdown` selector at some point.

That said, it's mostly just a style thing and if you don't like it it can certainly be changed.